api/services/orders/index.js

```diff
@@ -68,15 +69,15 @@ class OrdersService {
       );
     }
 
-    const amount = examinations.reduce(
-      (acc, curr, i) => acc + (i === 0 ? +curr.primaryPrice * 100 : +curr.secondaryPrice * 100),
-      0,
-    );
+    const event = await DB.events.findByPk(eventId);
+
+    const amount = event.primaryPrice * 100 + event.secondaryPrice * 100;
```

Comment:
```
amount = primaryPrice + ((numOfExams-1) * secondaryPrice)
```

Reply:
why do we want to `numOfExams-1`?
because student can choose only 1 exam as well?

Reply:
done

Reply:
First exam would be primary price rest would be secondary

Reply:
Basically
1. If student purchases 1 exam, amount = primary price
2. purchases 2 exams, amount = primary + secondary
3. purchases 3 exams, amount = primary + (2 * secondary)